src/ripple/app/ledger/LedgerTiming.cpp

```diff
@@ -128,7 +128,7 @@ bool ContinuousLedgerTiming::haveConsensus (
     }
 
     // If 80% of current proposers (plus us) agree on a set, we have consensus
-    if (((currentAgree * 100 + 100) / (currentProposers + 1)) > 80)
+    if (((currentAgree * 100 + 100) / (currentProposers + 1)) >= CONSENSUS_PCT)
```

Comment:
This formula also doesn't round up correctly. Fixed version:
```
if (((currentAgree + 1) * 100 + currentProposers) / (currentProposers + 1) >= CONSENSUS_PCT)
```

Its also not clear what we're gaining by abbreviating. Suggested: `CONSENSUS_PERCENT`.